code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -206,7 +214,11 @@
 		add_attack_logs(user, L, "stunned")
 	playsound(src, 'sound/weapons/egloves.ogg', 50, TRUE, -1)
 	deductcharge(hitcost)
+	return TRUE
 
+/obj/item/melee/baton/proc/baton_knockdown(mob/living/target, user_UID, knockdown_duration)
+	target.KnockDown(knockdown_duration)
+	REMOVE_TRAIT(target, TRAIT_WAS_BATONNED, "[user_UID]")
```

Comment:
the minor inconsistency between `, "[user_UID]")` here and `, user_UID)` above verly slightly concerns me, since it appears to be significant/intentional, but probably isn't?